chronicle/Models/SensorDataModel.xcdatamodeld/SensorDataModel.xcdatamodel/contents

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<model type="com.apple.IDECoreDataModeler.DataModel" documentVersion="1.0" lastSavedToolsVersion="19186" systemVersion="20G165" minimumToolsVersion="Automatic" sourceLanguage="Swift" userDefinedModelVersionIdentifier="">
+    <entity name="SensorData" representedClassName="SensorData" syncable="YES" codeGenerationType="class">
+        <attribute name="data" attributeType="Binary"/>
+        <attribute name="endTimestamp" optional="YES" attributeType="Date" usesScalarValueType="NO"/>
+        <attribute name="id" attributeType="UUID" usesScalarValueType="NO"/>
+        <attribute name="sensorType" attributeType="String"/>
+        <attribute name="startTimestamp" attributeType="Date" usesScalarValueType="NO"/>
+        <attribute name="timezone" attributeType="String"/>
+        <attribute name="writeTimestamp" attributeType="Date" usesScalarValueType="NO"/>
+    </entity>
+    <elements>
+        <element name="SensorData" positionX="-63" positionY="-18" width="128" height="134"/>
+    </elements>
+</model>
```

Comment:
missing newline

Reply:
This was automatically generated by xcode so there is no reason to modify it